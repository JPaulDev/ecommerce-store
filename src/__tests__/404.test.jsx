import renderer from 'react-test-renderer';
import NotFound from '../pages/404';

describe('NotFound component', () => {
  it('renders a 404 error page', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
