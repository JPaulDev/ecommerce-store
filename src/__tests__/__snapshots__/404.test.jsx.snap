// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders a 404 error page 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: white;
  border: 1px solid var(--color-grey-12);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-size: var(--font-size-18);
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-height: 600px;
  padding: 30px;
  text-align: center;
}

.c0 img {
  height: auto;
  max-width: 100%;
}

.c0 h1 {
  font-family: 'Uni Sans Book',sans-serif;
  font-size: var(--font-size-32);
  margin: 10px 0;
}

<div
  className="c0"
>
  <img
    alt=""
    data-nimg="future"
    decoding="async"
    height={24}
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=48&q=85"
    srcSet="/_next/image?url=%2Fimg.jpg&w=32&q=85 1x, /_next/image?url=%2Fimg.jpg&w=48&q=85 2x"
    style={
      {
        "color": "transparent",
      }
    }
    width={24}
  />
  <h1>
    Not Found
  </h1>
  <p>
    The requested page doesn‘t exist or you don‘t have access to it.
  </p>
</div>
`;
